tinygrad/shape/symbolic.py

```diff
@@ -194,6 +205,7 @@ def __mod__(self, b: Union[Node, int]):
   def get_bounds(self) -> Tuple[int, int]:
     return (self.a.min*self.b, self.a.max*self.b) if self.b >= 0 else (self.a.max*self.b, self.a.min*self.b)
   def substitute(self, var_vals: Dict[VariableOrNum, Node]) -> Node: return self.a.substitute(var_vals) * (self.b if isinstance(self.b, int) else self.b.substitute(var_vals))
+  def unbind(self) -> Tuple[Node, None]: return self.substitute({v: v.unbind()[0] for v in self.vars() if v.val is not None}), None
```

Comment:
Can this just be on the root class? I see it's copy pasted below

Reply:
done